source/frontend/src/components/common/form/FormGuide/FormGuideView.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FunctionComponent, PropsWithChildren } from 'react';
+import { Col, Collapse, Row } from 'react-bootstrap';
+import { FaInfoCircle } from 'react-icons/fa';
+import { MdArrowDropDown, MdArrowDropUp } from 'react-icons/md';
+import styled from 'styled-components';
+
+export interface IFormGuideViewProps {
+  title: string;
+  isCollapsed: boolean;
+  guideBody: React.ReactNode;
+  toggleCollapse: () => void;
+}
+
+const FormGuideView: FunctionComponent<PropsWithChildren<IFormGuideViewProps>> = ({
+  title,
+  isCollapsed,
+  guideBody,
+  toggleCollapse,
+}) => {
+  return (
+    <>
+      <StyledSectionHeader>
+        <StyledRow className="no-gutters" onClick={() => toggleCollapse()}>
+          <Col>
+            <StyledFaInfoCircle />
+            {title}
+          </Col>
+          <Col xs="1">
+            {isCollapsed && <ArrowDropDownIcon />}
+            {!isCollapsed && <ArrowDropUpIcon />}
+          </Col>
+        </StyledRow>
+      </StyledSectionHeader>
+
+      <Collapse in={!isCollapsed}>
+        <StyledSection>{guideBody}</StyledSection>
+      </Collapse>
+    </>
+  );
+};
+
+export default FormGuideView;
+
+const StyledSectionHeader = styled.h2`
+  font-size: 16px;
+  color: #1a5a96;
+  font-style: italic;
+  font-weight: bold;
+  text-decoration: none solid rgb(26, 90, 150);
```

Comment:
nit: would prefer if those colors "#1a5a96" and rgb(26, 90, 150) came from out CSS "theme"